tensorflow/python/ops/array_ops.py

```diff
@@ -100,19 +100,34 @@
 
 
 def rank(input, name=None):
-  """Override for gen_array_ops.rank(), making it work for SparseTensor.
+  """Returns the rank of a tensor.
+
+  This operation returns an integer representing the rank of `input`.
+
+  For example:
+
+  ```prettyprint
```

Comment:
I copied this from the generated file `gen_array_ops.py`, if it works there it should work here right?